sql/hive/src/test/scala/org/apache/spark/sql/hive/MetastoreDataSourcesSuite.scala

```diff
@@ -1104,4 +1104,22 @@ class MetastoreDataSourcesSuite extends QueryTest with SQLTestUtils with TestHiv
       }
     }
   }
+
+  test("CTAS: disallow create temporary table ... using ... as ...") {
```

Comment:
Can you move this test case to `HiveDDLCommandSuite`, which contains the Hive-specific parser test cases? You also can see the other CTAS test cases there. Thanks!